media-fonts/fantasque-sans-mono/fantasque-sans-mono-1.7.2.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2019 Gentoo Foundation
```

Comment:
Old copyright header, you should use repoman to commit so it fixes this for you automatically.

Reply:
This PR was created in 2019. But since this is going nowhere I'd probably just close it and keep things in my own overlay.

Reply:
We stopped using `Foundation`, repoman would flag and fix it for you automatically. I'd upated the year after merging, just wanted to point this out.

I was gonna take care of this PR now that I found it... devs not beloning to projects won't get noted about them either.

Reply:
Treat yourself, there is a open bug here: https://bugs.gentoo.org/702250